ts/Dashboards/Component/Component.ts

```diff
@@ -1001,14 +1055,20 @@ namespace Component {
     }
 
     /**
+     * Extract all components names from the registry.
      *
+     * @return {Array<string>}
+     * Returns the array of components names.
      */
     export function getAllComponentNames(): Array<string> {
```

Comment:
This `getAllComponentNames` isn't used anywhere. Do we need it?

Reply:
@goransle ?